source/extensions/formatter/metadata/metadata.h

```diff
@@ -16,17 +16,9 @@ namespace Formatter {
 class MetadataFormatterCommandParser : public ::Envoy::Formatter::CommandParser {
 public:
   MetadataFormatterCommandParser();
```

Comment:
Constructor should be =default or not declared at all now that the impl was removed.

Reply:
Hmmmmm, so weird. I need to re run the tests in my local env.

I remember I had fixed all these. Seems some code changes are lost.